src/JavaMultipleLineComment.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace AutoRest.Java.DanModel
+﻿namespace AutoRest.Java
 {
     public class JavaMultipleLineComment
```

Comment:
Seems like this is now specifically about generating Javadocs--would encourage calling it JavadocComment, especially as `/* */` is considered only a block comment while `/** */` is a Javadoc comment

Reply:
That's a really good idea. I'll rename these Comment classes in a future pull request.